nautobot/core/api/serializers.py

```diff
@@ -83,8 +83,8 @@ def fields(self):
                 # No available request?
                 return fields
 
-            # opt-in fields only applies on GET requests, for other methods we support these fields regardless
-            if request is not None and request.method != "GET":
+            # opt-in fields only applies on GET / OPTIONS requests, for other methods we support these fields regardless
+            if request is not None and request.method not in ["GET", "OPTIONS"]:
```

Comment:
Gets info about what the endpoint supports (e.g. encodings, choices-field values, that sort of thing). Mostly this was because I noticed that rendering the browsable API (e.g. accessing `/api/ipam/vrfs/` with a browser) actually instantiates the serializer for both GET and OPTIONS and it seemed weird to have OPTIONS still load the M2M fields and the like.

I'll revisit this and see if it's still needed/useful with my latest approach.